scripts/travis-functest.sh

```diff
@@ -27,21 +27,12 @@ sudo scripts/minikube.sh create-block-pool
 # pull docker images to speed up e2e
 sudo scripts/minikube.sh cephcsi
 sudo scripts/minikube.sh k8s-sidecar
-KUBE_MAJOR=$(kube_version 1)
-KUBE_MINOR=$(kube_version 2)
-# skip snapshot operation if kube version is less than 1.17.0
-if [[ "${KUBE_MAJOR}" -ge 1 ]] && [[ "${KUBE_MINOR}" -ge 17 ]]; then
-    # delete snapshot CRD created by ceph-csi in rook
-    scripts/install-snapshot.sh delete-crd
-    # install snapshot controller
-    scripts/install-snapshot.sh install
-fi
+# install snapshot controller and create snapshot CRD
+scripts/install-snapshot.sh install
```

Comment:
`scripts/install-snapshot.sh delete-crd` is not called anymore, can it be removed from the script?

Reply:
yeah that can be removed, done